client/components/domains/register-domain-step/index.jsx

```diff
@@ -274,7 +274,15 @@ const RegisterDomainStep = React.createClass( {
 		async.parallel(
 			[
 				callback => {
-					if ( ! domain.match( /.{3,}\..{2,}/ ) ) {
+					if ( !domain.match( /.{3,}\..{2,}/ ) ) {
```

Comment:
Oops. IDE misconfiguration can be bad sometimes. Fixed and fixed the linter to tell me about these.